portfolio/src/main/webapp/script.js

```diff
@@ -138,20 +138,26 @@ const mapStyle = [
   ];
 
 /**
- * Calls the methods that load the home page.
+ * Calls the methods that load the page.
  */
 async function start(page) {
   if (page === 'home') {
     loadChartsApi();  
     createMap();
     const response = await fetch('/login');
-    const loggedIn = await response.json();
-  
-    // Hide the comment section & nickname link and show text asking user to log in if they not logged in.
+    const info = await response.json();
+    const loggedIn = info.loggedIn;
+    const loginElement = document.getElementById('login');
+ 
+    // Show approprite log in/out prompt, and hide the comments & nickname link if user is logged out.
     if (loggedIn) {
+      loginElement.innerHTML = '<a>Log Out</a>';
```

Comment:
I think that since loginElement is already a `<a>` tag you can probably just do `loginElement.innerText = 'Log Out'` and don't need to worry about setting the `<a>` again (which could possibly erase some class names or other attributes)

Reply:
Fixed. Since I haven't given this tag a class or attributes, I don't think anything else can be simplified.